File: API_MOVIL/API_MOVIL/Models/ShippingLine.cs
using System;
using System.Collections.Generic;

namespace API_MOVIL.Models;

public partial class ShippingLine
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public int CityId { get; set; }

    public virtual City City { get; set; } = null!;
}

File: SIMEXWeb/backend/app/Models/ShippingLine.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

/**
 * Naviera (shipping line) asociada a la ciudad donde opera.
 *
 * @property int    $id
 * @property string $name
 * @property int    $city_id FK a `cities`.
 */
class ShippingLine extends Model
{
    use HasFactory;

    public $timestamps = false;

    protected $fillable = ['name', 'city_id'];

    /**
     * Ciudad base de la naviera.
     *
     * @return BelongsTo
     */
    public function city(): BelongsTo
    {
        return $this->belongsTo(City::class);
    }
}
